packages/arch/packages/popout/src/index.js

```diff
@@ -127,8 +133,9 @@ class Popout extends Component<Props, State> {
   render() {
     const { children, getModalRef, style, width } = this.props;
     let { leftOffset, topOffset, arrowLeftOffset } = this.state;
+    const attachTo = ((document.body: any): HTMLElement); // NOTE: flow junk
 
-    return document.body
+    return attachTo
       ? createPortal(
```

Comment:
Couldn't you remove the `attachTo ? createPortal(...) : null` and just make it `createPortal(...)`?

Reply:
Flow still gets cranky...